src/App.tsx

```diff
@@ -1,8 +1,72 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { Todo, User, PreparedTodos } from './interfaces';
+import { getTodos, getUsers } from './api';
+import { TodosList } from './components/TodosList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setPreparedTodos] = useState<PreparedTodos[]>([]);
```

Comment:
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```

Reply:
I have changed setPreparedTodos to setTodos,  but how I can rename <PreparedTodos[]> to <Todo[]>? If I do that, my app will crash because of PreparedTodos, it is a valid type of todos described in interfaces.